packages/codemod/bin/reka-ui-codemod.mjs

```
@@ -0,0 +1,141 @@
+  if (stats.isFile()) {
+    if (extensions.has(extname(path).slice(1)))
+      yield path
+    return
+  }
+  if (!stats.isDirectory())
+    return
+  const entries = readdirSync(path, { withFileTypes: true }).sort((a, b) => a.name.localeCompare(b.name))
+  for (const entry of entries) {
+    if (entry.isDirectory()) {
+      if (!SKIPPED_DIRECTORIES.has(entry.name))
+        yield * walk(join(path, entry.name), extensions)
```

ESLint

[error] Unexpected space before *.

(style/yield-star-spacing)